### DISCUSSION TITLE
Slack

### DISCUSSION THREAD

**Discussion by designnatix** [carbon-design-system/carbon#7831] [Help] [answered]
Can non-IBMers join your Slack community? It is mentioned on the contributing page.

https://github.com/carbon-design-system/carbon/blob/master/.github/CONTRIBUTING.md

**Comment by tw15egan** [comment, marked as answer]
The slack is for internal IBMers only, but we've created a public Discord where you can chat with the team. [Here's the link](https://discord.gg/3Y3VBpG97P)

**Reply by designnatix** [reply]
Thanks! I saw there aren't any posts in the Design channel on Discord. Is there a place to get answers to design questions? Can I post to the development channel on Discord?

**Reply by tw15egan** [reply]
Sure thing, we can pass along any design-related questions you may have to the team 👍🏻
